src/qtiItem/core/Loader.js

```diff
@@ -106,7 +107,7 @@ var Loader = Class.extend({
     getLoadedClasses() {
         return _.keys(this.qti);
     },
-    loadItemData(data, callback) {
+    loadItemData(data, callback, perInteractionRP) {
```

Comment:
I'm still uncomfortable with this additional parameter. I think it should no be needed, as the loader should be able to adapt gracefully without. In fact, it should natively recognize the response processing no matter the per-item or per-interaction scheme.

Reply:
without this parameter it is not possible to properly recognize response processing type